torch/onnx/_internal/exporter/_dynamic_shapes.py

```diff
@@ -39,6 +39,14 @@ def from_dynamic_axes_to_dynamic_shapes(
 
     Detail on Dim.DYNAMIC: `#133620 <https://github.com/pytorch/pytorch/pull/133620>`_
     """
+
+    warnings.warn(
+        "from_dynamic_axes_to_dynamic_shapes is deprecated and will be removed in a future release. "
```

Comment:
Describe the behavior instead? Since the function is not public.

Reply:
Should be more informative now